Lib/test/test_sax.py

```diff
@@ -1311,6 +1312,50 @@ def test_nsattrs_wattr(self):
         self.assertEqual(attrs.getQNameByName((ns_uri, "attr")), "ns:attr")
 
 
+# ===========================================================================
+#
+#   Sax parser property tests
+#
+#   Currently only tests the condition reported in issue bpo-6686
+#
+# ===========================================================================
+
+class PropertyContentHandler(XMLGenerator):
+    def __init__(self, test_harness, reader, *args, **kwargs):
+        test_harness.result = StringIO()
+        super().__init__(test_harness.result, *args,
+                         encoding='UTF-8', **kwargs)
+        self.test_harness = test_harness
+        self.reader = reader
+
+    def startElement(self, name, attr):
+        property_ = self.reader.getProperty(property_xml_string)
+        self.test_harness.assertIsInstance(property_, str)
+        super().startElement(name, attr)
+
+
+class SaxPropertyTest(unittest.TestCase):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.result = None
+
+    def test_xml_str_str(self):
+        reader = create_parser()
+        reader.setContentHandler(PropertyContentHandler(self, reader))
+        input_\
+            = '<?xml version="1.0" encoding="UTF-8"?>\n<test>Hi there</test>'
+        reader.parse(StringIO(input_))
+        self.assertEqual(input_, self.result.getvalue())
```

Comment:
Does this test the result of the `.getProperty(property_xml_string)` calls? That's what I'd like to see tested for more than its return type.

Reply:
By using XMLGenerator in the HandleContent hierarchy, the oiginal source is reconstructed.  This gives an indirect verification.

Reply:
Related to line breaks - I am following the recommendation in [Pep8](https://pep8.org/#break-before-or-after-binary-operator). It is recommended as the new approach.

Reply:
I am waiting to your reaction to my comments before I push the latest changes.